### PR TITLE
Update README.md to include instructions of building multi-arch JARs.…

### PR DESCRIPTION
**Pull request by geoffreyblake**
Update README.md Java section instructions with revised profiling instructions and how to build multi-arch JARs for those who are developing on both x86 and Graviton.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,22 +189,7 @@ C5.4XL	                        M6g.4XL
 ```
 
 ## Java
-For languages that rely on a JIT (such an Java), the symbol information that is
```
Comment (AGSaidi) [inline README.md:192]: Feels like a bit of intro text is needed to motivate multi-arch JARs and building them.

Comment (geoffreyblake) [reply README.md:192]: PR updated with some justification as to why multi-arch Jars are beneficial.

File: README.md [hunk 2]
```diff
@@ -214,14 +199,88 @@ without the JNI the performance can suffer.
...
+
+There is nominally a multi-step process to build the native shared objects for each supported
+architecture before doing the final packaging with Maven, SBT, Gradle etc. Below is an example
+of how to create your JAR using Maven that contains shared libraries for multiple distributions
+and architectures for running your Java application interchangeably on AWS EC2 instances
+based on x86 and Graviton processors:
+
+```
+# Create two build instances, one x86 and one Graviton instance.
+# Pick on instance to be the 'master' instance.
```
Comment (AGSaidi) [inline README.md:224]: maybe
```# Pick one instance to be the primary instance and the log into the secondary instance```

File: README.md [hunk 3]
```diff
@@ -214,14 +199,88 @@ without the JNI the performance can suffer.
...
+# Log into the non-master instance
+$ cd java-lib
+$ mvn package
+$ find target/ -name "*.so" -type f -print
+
+# Note the directory this so file is in, it will be in a directory
+# such as: target/classes/org/your/class/hierarchy/native/OS/ARCH/lib.so
+
+# Log into the master build instance
```
Comment (AGSaidi) [inline README.md:233]: ``` #Log into the primary instance```

File: README.md [hunk 4]
```diff
@@ -214,14 +199,88 @@ without the JNI the performance can suffer.
...
+$ mvn package
+
+# To do a release to Maven Central and/or Sonatype Nexus:
+$ mvn release:prepare
+$ mvn release:perform
+
+# When creating a single Jar for all platform native libraries, 
+# the release plugin's configuration must be modified to specify 
+# the plugin's `preparationGoals` to not include the clean goal.
+# See http://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html#preparationGoals
```
Comment (AGSaidi) [inline README.md:252]: This seems to come pretty late... I imagine someone would execute all the steps and then be like ohh.. it didn't work.

File: README.md [hunk 5]
```diff
@@ -214,14 +199,88 @@ without the JNI the performance can suffer.
...
+# For more details.
+
+```
+
+This is one way to do the JAR packaging with all the libraries in a single Jar.  It is also possible to build a Jar for each supported
+platform and use something like the nar maven plugin to manage each specific Jar.  To build all the Jars, we recommend to build on native
```
Comment (AGSaidi) [inline README.md:258]: nar a word?

Comment (geoffreyblake) [reply README.md:258]: Yeah, its a plugin for maven to manage multiple native libraries.  Forgot to make it a link.

File: README.md (hunk 4 above)
Comment (geoffreyblake) [reply README.md:249]: Moved this up.

File: README.md [hunk 6]
```diff
@@ -206,38 +206,102 @@ C5.4XL	                        M6g.4XL
...
+any of the resulting files are shared-objects and if an aarch64 (arm64) shared-object is missing:
+```
+$ jar xf foo.jar
+$ find . -name "*.so" | xargs file
+```
+For each x86-64 ELF file, check if there is a corresponding aarch64 ELF file
+in the binaries. With some common packages \(e.g. commons-crypto\) we've seen that
+even though a JAR can be built supporting Arm manually, artifact repositories such as
+[Maven - Central](https://mvnrepository.com/repos/central) do not have updated versions.
+Feel free to open an issue in this GitHub repo for advice.
```
Comment (AGSaidi) [inline README.md:226]: sounds random here. advice for what?

Comment (AGSaidi) [reply README.md:226]: the other CR fixes this.

**Pull request merged**